### PR TITLE
{bio}[intel/2019a] Biopython v1.73 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3126.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/416fc6e34581c086b53715b4bf74f388 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2053.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c76d2504a8f033f36cf0fb61122edd4a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2658.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fad7a2736a3a5e859dfaf91553fb6b9a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3162.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6c0349ffd2f8a32e5042edd3806fd370 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2099.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6b9a8462a394f426f9c65b683b31b6bc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2658.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7dca464e5a163b3631a385bcc5fff353 for a full test report.

File: easybuild/easyconfigs/b/Biopython/Biopython-1.73-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+source_urls = ['http://biopython.org/DIST']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['70c5cc27dc61c23d18bb33b6d38d70edc4b926033aea3b7434737c731c94a5e0']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [
+    ('SciPy-bundle', '2019.03'),  # for nuumpy
```
Comment (vanzod) [inline easybuild/easyconfigs/b/Biopython/Biopython-1.73-intel-2019a.eb]: @boegel Tiny typo in the comment here...

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/463ae74ddc1775544dd337d60c9f5d0d for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**